src/chapters/data-viz/data-viz.rst

```diff
@@ -1,33 +1,36 @@
 Data Visualization with Python
 ==============================
 
-Readings
---------
+Read the following articles, follow along where instructed:
 
-| `An Intuitive Guide to Visualization in Python <https://www.analyticsvidhya.com/blog/2021/02/an-intuitive-guide-to-visualization-in-python/>`__. 
+Intro to Data Vizualization
+---------------------------
 
-| `What is Data Visualization and Why Is It Important? <https://www.import.io/post/what-is-data-visualization/>`__.
+1. `An Intuitive Guide to Visualization in Python <https://www.analyticsvidhya.com/blog/2021/02/an-intuitive-guide-to-visualization-in-python/>`__. 
 
-| `10 Essential Data Visualization Techniques, Concepts & Methods To Improve Your Business - Fast <https://www.datapine.com/blog/data-visualization-techniques-concepts-and-methods/>`__.
+2. `What is Data Visualization and Why Is It Important? <https://www.import.io/post/what-is-data-visualization/>`__.
 
-| `Data Visualization 101: How to Choose the Right Chart or Graph for Your Data <https://blog.hubspot.com/marketing/types-of-graphs-for-data-visualization>`__.
+3. `10 Essential Data Visualization Techniques, Concepts & Methods To Improve Your Business - Fast <https://www.datapine.com/blog/data-visualization-techniques-concepts-and-methods/>`__.
 
-| `Designing Charts and Graphs: How to Choose the Right Data Visualization Types <https://www.datapine.com/blog/how-to-choose-the-right-data-visualization-types/>`__. 
+4. `Data Visualization 101: How to Choose the Right Chart or Graph for Your Data <https://blog.hubspot.com/marketing/types-of-graphs-for-data-visualization>`__.
 
-**Additional Resources and Practice**
+5. `Designing Charts and Graphs: How to Choose the Right Data Visualization Types <https://www.datapine.com/blog/how-to-choose-the-right-data-visualization-types/>`__. 
 
-| `Matplotlib Tutorial <https://www.w3schools.com/python/matplotlib_intro.asp>`__. 
+Matplotlib and Graphs
+---------------------
 
-| `Types of Graphs <https://www.jmp.com/en_au/statistics-knowledge-portal/exploratory-data-analysis/types-of-graphs.html#75af8421-08c4-4cdd-95c9-8197a372ad04>`__. 
+6. `Matplotlib Tutorial <https://www.w3schools.com/python/matplotlib_intro.asp>`__. 
+
+7. `Types of Graphs <https://www.jmp.com/en_au/statistics-knowledge-portal/exploratory-data-analysis/types-of-graphs.html#75af8421-08c4-4cdd-95c9-8197a372ad04>`__. 
```

Comment:
Under each new section ordered lists should restart and not continue from the previous section.

Use #. in front of link insted of hard coding.